content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -1,85 +1,72 @@
 ---
-title: "Example: Deploying Cassandra with Stateful Sets"
+title: "Example: Deploying Cassandra with a StatefulSet"
 reviewers:
 - ahmetb
 content_template: templates/tutorial
 weight: 30
 ---
 
 {{% capture overview %}}
-This tutorial shows you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this example, a custom Cassandra *SeedProvider* enables Cassandra to discover new Cassandra nodes as they join the cluster.
+This tutorial shows you how to run [Apache Cassandra](http://cassandra.apache.org/) on Kubernetes. Cassandra, a database, needs persistent storage to provide data durability (application _state_). In this example, a custom Cassandra seed provider lets the database discover new Cassandra instances as they join the Cassandra cluster.
```

Comment:
Possible edit:
This tutorial shows you how to run the [Apache Cassandra](...) database on Kubernetes. Cassandra
requires persistent storage to provide data durability.
- Check the link to Cassandra
- Possibly link to the seed provider code here so that readers know what it is.

Reply:
I have checked the link to Cassandra. http://cassandra.apache.org/ works.

I'm happy to leave the seed provider link tidying to a future pull request.